extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/internal/handler/ZonePlayerHandler.java

```diff
@@ -119,11 +121,6 @@
      */
     private final Object notificationLock = new Object();
 
-    /**
-     * Separate sound volume used for the notification
-     */
-    private String notificationSoundVolume = null;
```

Comment:
TODO: updateChannel in initialize of handler after restart

Reply:
Good that you found this one by yourself :) While reading your source I was already asking myself where the initial setting should come from since `handleConfigurationUpdate` is only called if the configuration is changed, right? If it is also called while starting the handler to load the intial config you can omit the implementation in `initialize`. Please check that first.

Reply:
This is not relevant anymore.